modules/platform/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/java/org/nuxeo/ecm/platform/audit/io/LogEntryJsonWriter.java

```diff
@@ -147,20 +148,15 @@ protected void writeExtendedInfo(JsonGenerator jg, String key, Serializable valu
             jg.writeObjectField(key, value);
         } else if (Map.class.isAssignableFrom(clazz)) {
             @SuppressWarnings("unchecked")
-            Map<String, Serializable> map = //
-                    ((Map<String, Serializable>) value).entrySet()
-                                                       .stream()
-                                                       // keep primitive values
-                                                       // blobs can be in WorkflowVariables for instance
-                                                       .filter(entry -> entry.getValue() != null
-                                                               && ClassUtils.isPrimitiveOrWrapper(
-                                                                       entry.getValue().getClass()))
-                                                       .collect(Collectors.toMap(Entry::getKey, Entry::getValue));
-            jg.writeObjectField(key, map);
+            Map<String,Serializable> map = (Map<String, Serializable>) value;
+            jg.writeObjectFieldStart(key);
+            for(Entry<String,Serializable> entry : map.entrySet()) {
+                writeExtendedInfo(jg,entry.getKey(),entry.getValue());
```

Comment:
Space before `(` and after `,` here an in several other places

Reply:
done